hack/local_setup.sh

```diff
@@ -0,0 +1,152 @@
+# THE SCRIPT SHOULD BE RUN WHILE IN THE MCM REPOSITRY IN HACK FOLDER & IT ASSUMES THAT IN THE PARENT DIRECTORY OF MCM
```

Comment:
whenver we create a new file , we need to add the license boilerplate by SAP
refer the other files in `hack` folder for this